score-data-format/src/main/java/io/cloudslang/content/json/utils/StringUtils.java

```diff
@@ -0,0 +1,15 @@
+package io.cloudslang.content.json.utils;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+/**
+ * Created by Folea Ilie Cristian on 2/5/2016.
+ */
+public class StringUtils {
+    public static String toString(Throwable e) {
+        StringWriter writer = new StringWriter();
+        e.printStackTrace(new PrintWriter(writer));
```

Comment:
maybe close resources before returning, or you could use one of the existing StringUtils that are already written in the other modules?